docs/Agent/installation/index.rst

```diff
@@ -0,0 +1,12 @@
+Installation
+=========================
+
+Choose any one approach to setup `Pbench Agent` on our host.  
```

Comment:
This is a general and important "style" issue for documentation. How do we want to define the "voice" of our documentation? This sentence acts as if the documentation voice is an active participant in the reader's activity: "_our_" host, rather than "your" host or "the system on which the agent code is deployed" (more passive and detached).

We need a conscious decision about defining that voice, and perhaps there are Red Hat standards we should consider?

Reply:
I have rephrased the sentence in the latest commit, could you please review that.
Thanks